test/suite/test_rollback_to_stable04.py

```diff
@@ -40,11 +41,25 @@ def mod_val(value, char, location, nbytes=1):
 # Test that rollback to stable always replaces the on-disk value with a full update
 # from the history store.
 class test_rollback_to_stable04(test_rollback_to_stable_base):
-    conn_config = 'cache_size=50MB,log=(enabled),statistics=(all)'
     session_config = 'isolation=snapshot'
 
+    in_memory_values = [
+        ('no_inmem', dict(in_memory=False)),
+        ('inmem', dict(in_memory=True))
+    ]
+
+    scenarios = make_scenarios(in_memory_values)
+
+    def conn_config(self):
+        config = ''
+        if self.in_memory:
+            config += 'cache_size=250MB,statistics=(all),in_memory=true'
+        else:
+            config += 'cache_size=20MB,statistics=(all),log=(enabled),in_memory=false'
+        return config
+
     def test_rollback_to_stable(self):
-        nrows = 10000
+        nrows = 1000
```

Comment:
Just a FYI - I'm guessing you reduced this to make the test run faster but when I originally wrote the test I found that failures only showed up at the higher row count.

Reply:
Yes, I reduced the rows to improve the test speed. Now I reverted it to original value with increased cache size, so the test run time has reduced.